DemoApp/DemoApp/Views/CardSamplePage.xaml

```diff
@@ -124,8 +124,7 @@
           <!-- Footer -->
           <StackLayout BackgroundColor="Transparent"
                        Padding="0,10,0,10">
-            <modus:TMButton Text="Button"
```

Comment:
In this page, can you replace the Switch with 'TMSwitch'. Default's switch's UI is bad in iOS

Reply:
Replaced switch modus switch